Move secondary requirement texts out of the background (#2060)
act-rules/act-rules.github.io: _rules/aria-attr-defined-5f99a7.md

  This rule checks that each `aria-` attribute specified is defined in ARIA 1.2.
accessibility_requirements:
  wcag20:1.3.1: # Info and Relationships (A)

>     secondary: This success criterion is **less strict** than this rule. This is because the rule does not ignore irrelevant ARIA properties. This is why some of the failed examples may not satisfy this success criterion.

Author: ```suggestion
    secondary: This success criterion is **less strict** than this rule. This is because the rule does not ignore irrelevant ARIA properties. Some of the failed examples satisfy this success criterion.
```

Reviewer: @WilcoFiers and others ... I would want us to define the "some" where as you'd want to map to what are the "some" examples that people can reference in context of what fails / passes as secondary reqs. Thanks!

Reviewer 2: ```suggestion
secondary: This success criterion is less strict than this rule. This is because the rule does not ignore irrelevant ARIA properties. Additionally, some of the failed examples meet this success criterion.
```

Reviewer 3: * I agree that the second sentence should be "warning, some failed examples pass the SC" rather than "warning, some failed examples fail the SC", because the later is the normal behaviour of rules.
* I'd rather keep some emphasis on "less strict" (like the bolding here), to show this is some sort of keyword. I'd even push toward having one such emphasised term in each secondary explanation since it can help us (and readers) categorise them ("less strict", "more strict", "partial", …) Once we get a bit more usage on these secondary requirements, we may even include these terms in the ACt rules format (or a similar document), with detailed explanation about what they mean.
*  agree that the "this is why" is a bit of a mouthful after a sentence full of "thus rule" and "this SC". Another possibility could be:
    > Therefore, some FE do satisfy the SC.
* I'm also not a huge fan of "may (not)" in the "some (passed/failed) example may (not) satisfy the SC". I feel it collides a bit with the RFC 2119 usage of these terms which is somewhat different here. This sounds like an instruction to rule writers (or part of the format): "when a SC is **less strict**, you **MAY** write FE that satisfy this SC"; while here we describe a more static state: "there exist FE that satisfy the SC, deal with it".
  (plus "may not satisfy" is always slightly ambiguous between "(may not) satisfy" (= are forbidden to satisfy) and "may (not satisfy)" (=are not forced to satisfy); "do not satisfy" doesn't have this ambiguity)

>     secondary: This success criterion is **less strict** than this rule. This is because the rule does not ignore irrelevant ARIA properties. This is why some of the failed examples may not satisfy this success criterion.

Reviewer 4: This rule is **more strict** than the success criterion because it does not ignore irrelevant ARIA properties. Therefore, some of the reported failures can be false positives rather than a success criterion failure.

My comment carries to the rest of the examples where this text appears. It was a little difficult to read and understand, as written.

Reviewer 3: I'm not a huge fan of using the term "false positives" here, since it has a connotation of "the rule is bad and generate false positives" while we want to carry the information "this rule is designed for a slightly different use case and tangentially affects this use case".

Typically, in a color contrast case, saying that the 1.4.6 rule "reports false positives (for 1.4.3)" is technically true but sounds a bit dramatic.

Reviewer 5: I like the change from saying "the rule more strict" (than the SC)  to "the SC is less strict" (than the rule).
